ledger/ledger-common/src/main/kotlin/net/corda/v5/ledger/common/transactions/LedgerTransaction.kt

```diff
@@ -0,0 +1,11 @@
+package net.corda.v5.ledger.common.transactions
+
+import net.corda.v5.base.annotations.DoNotImplement
+import net.corda.v5.crypto.SecureHash
+import java.security.PublicKey
+
+@DoNotImplement
+interface LedgerTransaction {
```

Comment:
I am not sure we should create common interfaces at this stage. I think it is quite limiting to force ledgers to have any commonality at the data model level. Lets just stick to Consensual ledger

Reply:
OK. I have removed the common one.